content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,166 @@
+In 2020 Q2 the project has seen **21% growth** in pull requests numbers compared to 2019 Q2, bots excluded.
+
+
+One may say that the Jenkins project already has everything needed to succeed.
+It is a result of continuous work by many community members,
+and this work will never end as long as the project remains active.
+Like in any other industry, the CI/CD ecosystem changes every day and sets new expectations from the automation tools in this domain.
+Same as the tools, the open source communities need to evolve so that they can address expectations, and onboard more users and contributors.
+CDF graduation process helped us to discover opportunities for improvement,
+and to address them.
+We reviewed the project processes and to compare them with the Graduated Project criteria defined in the link:https://github.com/cdfoundation/toc/blob/master/PROJECT_LIFECYCLE.md[CDF project lifecycle].
+Based on this review, we have made changes in our processes and documentation.
+It should improve experience of Jenkins users,
+and help to make the Jenkins community more welcoming to existing and newcomer contributors.
+
+== What changed for the project?
+
+Below you can find a few key changes we have applied during the graduation process:
+
+Public roadmap::
+
+We introduced a new link:/project/roadmap/[public roadmap] for the Jenkins project.
+This roadmap aggregates key initiatives in all community areas: features, infrastructure, documentation, community, etc.
+It makes the project more transparent to all Jenkins users and adopters,
+and at the same time helps potential contributors to find out the hot areas and opportunities for contribution.
+The roadmap is driven by the Jenkins community and it has a fully public process documented in jep:14[].
+
+More details about the public roadmap are coming next week, stay tuned for a separate blogpost.
+On July 10th we had an online contributor meetup about it,
+and you can find more information about the roadmap in its materials
+(link:https://docs.google.com/presentation/d/1_T2nZhP1WS2Fw0OLVAJV14Ke6nEsqBjLcdAHiygCmNs/edit?usp=sharing[slides], link:https://www.youtube.com/watch?v=ldWBY5BdQ5I[video recording]).
+
+User Documentation::
+
+* Jenkins Weekly Release line is now documented on our website (link:/download/weekly/[here]).
+  We have also reworked the downloads page and added guidelines explaining how to verify downloads.
+* A new link:/project/adopters[list of Jenkins adopters] was introduced on jenkins.io.
+  This list highlights Jenkins users and refers their case studies and success stories,
+  including ones submitted through the link:/blog/2020/04/30/jenkins-is-the-way/[Jenkins Is The Way] portal.
+  Please do not hesitate to add your companies there!
+
+Community::
+
+* We passed the link:https://www.coreinfrastructure.org/[Core Infrastructure Initiative (CII)] certification.
+  This certification helped us to verify compliance with open source best practices,
```

Comment:
Since we're continuing to review the certification, I think it is OK to use the present tense rather than past tense.  That is then consistent with the rest of the paragraph.
```suggestion
  This certification helps us to verify compliance with open source best practices,
```